Resources/public/mapbender.element.coordinatesutility.js

```diff
@@ -502,12 +502,17 @@
                 this.highlightLayer.olLayer.addFeatures(this.feature);
 
                 var lonLat = new OpenLayers.LonLat(this.lon, this.lat);
+                var zoomlevel = this.options.zoomlevel;
+                var olMapMaxZoom = this.mbMap.map.olMap.numZoomLevels;
```

Comment:
This is a little baffling in conjunction with the following `<` check. IMO olMapMaxZoom should be numZoomLevels - 1 (because that is the highest number one can set), and the following check should be <=.